{{cookiecutter.github_repository_name}}/docker-compose.yml

```diff
@@ -27,3 +38,11 @@ services:
       - ./:/code
     ports:
       - "8001:8001"
+    networks:
+      - backend
+ 
+volumes:
+  db-data:
+
+networks:
+  backend:
```

Comment:
hey @vsumner thanks for adding this! ✨ 

Why did you add a network for backend and a specific volume for postgres?

Reply:
Thats not required. I use it to keep my test data between down and ups. 
You can then destroy the data with [`docker-compose down -v`](https://docs.docker.com/compose/reference/down/).

Reply:
The backend isn't required for that, the volume is.

I don't remember if the backend is needed.